organizations.json

```diff
@@ -1861,10 +1861,10 @@
     },
     {
         "name": "Code for Tallahassee",
-        "website": "http://www.codefortallahassee.org/",
-        "events_url": "http://www.codefortallahassee.org/#events",
+        "website": "",
+        "events_url": "https://www.meetup.com/meetup-group-kFdPKltD/",
```

Comment:
It looks like our Meetup account rep was able to reclaim the `code-for-tallahassee` name for your meetup. Want to update this to use it?